examples/life-expectancy/life-expectancy/le_utils/utils.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import math
+
+import numpy as np
+
+
+def build_insert_query(row, cols, output_table):
```

Comment:
I think this is not necessary anymore, right? If not could you remove it?

Reply:
Done!